packages/augur-ui/src/modules/create-market/components/form-details.tsx

```diff
@@ -172,12 +169,12 @@ export default class FormDetails extends React.Component<
               focused={s.timeFocused}
               errorMessage={validations[currentStep].hour}
             />
-            <TimezoneDropdown onChange={(value: number, offset: string) => {
-              onChange("offsetName", value);
+            <TimezoneDropdown onChange={(offsetName: string, offset: number, timezone: string) => {
               onChange("offset", offset)
-            }} timestamp={endTime} timezone={offsetName} />
+              onChange("timezone", timezone)
```

Comment:
i don't see timezone being used at all?

Reply:
it is on my branch and everything is checked in. I'll see

Reply:
everything is checked in
![image](https://user-images.githubusercontent.com/3970376/62721142-34ffe600-b9d1-11e9-8664-8e88a2338f45.png)

Reply:
but then timezone is never used anywhere? don't see why it needs to get set

Reply:
you don't need both offsetName and timezone

Reply:
timezone is needed only to pick the correct item in the timezone dropdown. True, only the parts of the selected timezone is used.